src/tools/dbg/quictypes.h

```diff
@@ -1440,6 +1453,24 @@ struct QuicLibrary : Struct {
         return ReadType<UINT8>("PartitionCount");
     }
 
+    UINT64 CurrentHandshakeMemoryUsage() {
+        return ReadType<UINT64>("CurrentHandshakeMemoryUsage");
+    }
+
+    UINT64 TotalMemory() {
+        UINT64 QuicTotalMemory;
+        ReadTypeAtAddr<UINT64>(GetExpression("msquic!QuicTotalMemory"), &QuicTotalMemory);
+        return QuicTotalMemory;
+    }
+
+    UINT64 RetryHandshakeMemoryLimit() {
+        return (GetSettings().RetryMemoryLimit() * TotalMemory()) / UINT16_MAX;
```

Comment:
why divide by 64k?

Reply:
See https://github.com/microsoft/msquic/blob/main/src/core/binding.c#L1201. RetryMemoryLimit is a fracton of UINT16_MAX.

Reply:
Ah, makes sense